app/views/public/tower.html.erb

```diff
@@ -1,8 +0,0 @@
-<h1>Solana <%= params[:network].capitalize %> Tower</h1>
-<div class="row">
-  <div class="col-sm-12">
-    <p>Validators are sorted by block height, then active stake.</p>
-    <%= render 'tower_table', validators: @tower_leaders %>
```

Comment:
I think we can also delete `views/public/_tower_table.html.erb` file

Reply:
Thanks, I missed that